integration/zero/create.clj

```diff
@@ -0,0 +1,63 @@
+(ns zero.create
```

Comment:
These integration tests (create, start, etc) have been moved to `integration/zero/v1_endpoint_test.clj`

Reply:
Rebases kept resurrecting them. I believe they are gone now.